MigrationV3V4/Node/Skills/v3-skill-bot/package.json

```diff
@@ -14,6 +14,7 @@
     "busboy": "^0.2.13",
     "chrono-node": "^1.2.5",
     "dotenv": "^8.2.0",
+    "dotenv-extended": "^2.7.1",
```

Comment:
are we using both dotenv and dot-envextended in this project?  (or, does one require the other?)

Reply:
removed. not sure why it was included; it's not needed so long as we change `load()` to `config()`, which I did